utils/ssh.py

```diff
@@ -197,20 +201,52 @@ def get_transport(self, *args, **kwargs):
             self.connect()
         return super(SSHClient, self).get_transport(*args, **kwargs)
 
-    def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False, ensure_host=False):
+    def run_command(
+            self, command, timeout=RUNCMD_TIMEOUT, reraise=False, ensure_host=False,
+            ensure_user=False):
+        """Run a command over SSH.
+
+        Args:
+            command: The command. Supports taking dicts as version picking.
+            timeout: Timeout after which the command execution fails.
+            reraise: Does not muffle the paramiko exceptions in the log.
+            ensure_host: Ensure that the command is run on the machine with the IP given, not any
+                container or such that we might be using by default.
+            ensure_user: Ensure that the command is run as the user we logged in, so in case we are
```

Comment:
There is no specific usecase yet but I want to be sure that we can also use the login user if necessary and do not cut ourselves away from it.